src/lib/pipeline.ml

```diff
@@ -158,7 +158,7 @@ module Construct = struct
 
   let star fastq = Star fastq
 
-  let hisat fastq = Hisat fastq
+  let hisat ?(configuration = Hisat.Configuration.default_v1) fastq = Hisat(configuration, fastq)
```

Comment:
Actually maybe :)

I don't know how the two versions are different, if you think users expect to very different results then yes `v1` is better.